modules/http-client/src/main/java/com/devonfw/module/httpclient/common/impl/AbstractSyncServiceHttpClient.java

```diff
@@ -0,0 +1,76 @@
+
+package com.devonfw.module.httpclient.common.impl;
+
+import java.net.http.HttpResponse;
+
+import com.devonfw.module.service.common.api.client.SyncServiceClient;
+import com.devonfw.module.service.common.api.client.async.ServiceClientInvocation;
+import com.devonfw.module.service.common.api.client.async.ServiceClientStub;
+import com.devonfw.module.service.common.base.client.AbstractSyncServiceClient;
+
+/**
+ * Abstract base implementation of {@link SyncServiceClient} using Java HTTP client.
+ *
+ * @param <S> type of the {@link #get() service client}.
+ * @param <F> type of the owning {@link SyncServiceClientFactoryHttp factory}.
+ * @since 2021.08.003
+ */
+
```

Comment:
can you remove all these pointless empty lines between JavaDoc and Signature, please?